cunumeric/array.py

```diff
@@ -390,13 +289,11 @@ def __init__(
         else:
             self._thunk = thunk
         self._legate_data: Union[dict[str, Any], None] = None
-        # TODO: flags are passed. We use this argument only for views
-        # Not all routines are changed to pass these arguments correctly yet.
-        if flags is not None:
-            self._flags = flagsobj.copy(flags)
-            self._flags._owndata = False
```

Comment:
Not sure what we want to do here, `owndata` is not settable on the actual np `flagsobj`

Reply:
skipping for now will create follow-up GH issue